template-tags.php

```diff
@@ -472,6 +473,17 @@ function coauthors_wp_list_authors( $args = array() ) {
 	}
 
 	$authors = apply_filters( 'coauthors_wp_list_authors_array', $authors );
+	
+	// only show guest authors if the $args is set to true
+	if ( $args['guest_authors_only'] ) {
+		$guest_authors = [];
+		foreach ( $authors as $author ) {
```

Comment:
Rather than do another foreach here, could we instead skip adding authors to the array within the foreach at [line 464](https://github.com/rebeccahum/Co-Authors-Plus/blob/9b57a4344c75e0825e5a30118721ae6a9f94cef9/template-tags.php#L464)